config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
-  get '/access-requests' => 'access_requests#index'
-  get '/organisations' => 'organisations#index'
+  root to: redirect('/organisations-engagement-report')
```

Comment:
No need to redirect `root to: 'reports#show_organisations_engagement_report'`

Reply:
Actually this is deliberate – I'd rather have the redirect and have the report on one URL only...